src/org/usfirst/frc2876/Steamworks2017/subsystems/DriveTrain.java

```diff
@@ -229,6 +229,12 @@ public void velocityDistanceStraight() {
 		rightMaster.set(r);
 		leftMaster.set(l);
 	}
+	
+	public void velocityDistance() {

```

Comment:
I don't think this will work for small distances.  The rpm output will be too small to move the robot to overcome friction/inertia.